src/Components/Calculator.js

```diff
@@ -1,31 +1,87 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable react/destructuring-assignment */
 import React, { Component } from 'react';
+import calculate from '../Logic/Calculate';
 
-// eslint-disable-next-line react/prefer-stateless-function
 export default class calculator extends Component {
+  constructor() {
+    super();
+    this.state = { total: '0', next: null, operation: null };
+  }
+
+  changeMessege(name) {
+    const result = calculate(this.state, name);
+    // eslint-disable-next-line react/no-access-state-in-setstate
```

Comment:
- [ ] please avoid turning off any linters, linters are used to help you make your code work better and be more readable.